lang/cs/Org.Apache.REEF.Tests/Functional/RuntimeName/RuntimeNameTest.cs

```diff
@@ -33,27 +37,76 @@ namespace Org.Apache.REEF.Tests.Functional.Driver
     public class RuntimeNameTest : ReefFunctionalTest
```

Comment:
Why is it necessary to make these functional tests? Couldn't we test the functionality without launching a REEF job.